styles/styles.css

```
@@ -1,7 +1,290 @@
-@import url("./colors.css");
-@import url("./fonts.css");
-
-.test {
-  color: var(--color-primary);
-  font-family: var(--font-primary);
-}
+@import url("./colors.css");
+@import url("./fonts.css");
+
+/* ==== COMMON ==== */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: var(--color-primary);
+  font-family: var(--font-primary);
+}

```

Te falto definir --color-primary y --font-primary, pero se que daniel ya los definio, puedes eliminar este pedazo


```suggestion
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```